reactClient/src/Components/searchComponents/DynamicCards.js

```diff
@@ -43,7 +43,7 @@ import {
 									/>
 									<CardContent>
 										<Typography  gutterBottom>
-										<img src={require('../../img/stockdormimage.jpg')} width="80%"  alt/>
+										<img src={require('../../img/stockdormimage.jpg')} width="80%"  alt=""/>
 										</Typography>
```

Comment:
This was also throwing errors about assigning a boolean value to alt. So I set it to an empty string.